src/index.html

```diff
@@ -12,7 +12,40 @@
       href="styles/index.scss"
     />
```

Comment:
The link to the stylesheet should reference a CSS file, not a SCSS file. SCSS files need to be compiled into CSS. Please ensure you link to the compiled CSS file instead.